From 7552b8d49901b1f55e159e5b232cdec10abcf65d Mon Sep 17 00:00:00 2001
From: Radostin Stoyanov <rst0git@users.noreply.github.com>
Date: Tue, 3 Apr 2018 04:06:53 +0100
Subject: [PATCH] man: machinectl: update fedora exmple URL (#8642)

(cherry picked from commit fcc7ce4c8e68b0eaef2db1649d1b40ad54f8dc9c)
---
 man/machinectl.xml | 8 ++++----
 1 file changed, 4 insertions(+), 4 deletions(-)

diff --git a/man/machinectl.xml b/man/machinectl.xml
index 1adebf5da0..7bc9ee208e 100644
--- a/man/machinectl.xml
+++ b/man/machinectl.xml
@@ -999,12 +999,12 @@
       <title>Download a Fedora image, set a root password in it, start
       it as service</title>
 
-      <programlisting># machinectl pull-raw --verify=no https://dl.fedoraproject.org/pub/fedora/linux/releases/23/Cloud/x86_64/Images/Fedora-Cloud-Base-23-20151030.x86_64.raw.xz
-# systemd-nspawn -M Fedora-Cloud-Base-23-20151030
+      <programlisting># machinectl pull-raw --verify=no https://dl.fedoraproject.org/pub/fedora/linux/releases/27/CloudImages/x86_64/images/Fedora-Cloud-Base-27-1.6.x86_64.raw.xz
+# systemd-nspawn -M Fedora-Cloud-Base-27-1.6.x86_64
 # passwd
 # exit
-# machinectl start Fedora-Cloud-Base-23-20151030
-# machinectl login Fedora-Cloud-Base-23-20151030</programlisting>
+# machinectl start Fedora-Cloud-Base-27-1.6.x86_64
+# machinectl login Fedora-Cloud-Base-27-1.6.x86_64</programlisting>
 
       <para>This downloads the specified <filename>.raw</filename>
       image with verification disabled. Then, a shell is opened in it
-- 
2.17.0

